python/ray/tune/syncer.py

```diff
@@ -677,24 +679,39 @@ def __getstate__(self):
 
 
 class _DefaultSyncer(_BackgroundSyncer):
-    """Default syncer between local storage and remote URI."""
+    """Default syncer between local and remote storage, using `pyarrow.fs.copy_files`"""
```

Comment:
Shall we make a new Syncer class for storage context that doesn't depend on the legacy remote_storage utils?